.meta/vercel.json

```diff
@@ -0,0 +1,7 @@
+{
+  "buildCommand": null,
+  "devCommand": null,
+  "framework": null,
+  "installCommand": null,
+  "trailingSlash": true
```

Comment:
Without this, if you visit a page without a trailing slash that has assets, such as the unit test coverage report, it will incorrectly try to grab the assets from the root instead of relative, e.g. `/asset.js` instead of `/unit-test-coverage/asset.js`